autopayroll-CordaService/contracts/src/main/kotlin/com/autopayroll/contracts/MoneyStateContract.kt

```diff
@@ -0,0 +1,26 @@
+package com.autopayroll.contracts
+
```

Comment:
Why are you not using the net.corda package like most of the other samples?

Reply:
What do you mean?

Reply:
I just picked 2 samples at random and they are using the `net.corda.examples` package but you are using the `com.autopayroll` package.
https://github.com/corda/samples/tree/release-V4/blacklist/src/main/kotlin/net/corda/examples/attachments
https://github.com/corda/samples/tree/release-V4/obligation-cordapp/kotlin-source/src/main/kotlin/net/corda/examples/obligation

For consistency it might be better to use the sample package across the samples.